string_permutation_problem/permute.py

```diff
@@ -0,0 +1,22 @@
+def permute_string(my_str):
+    string_list = []
+    # This is a recursive solution to the string permutation problem.
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpncyce3tu/string_permutation_problem/permute.py
+++ b/tmp/tmpncyce3tu/string_permutation_problem/permute.py
@@ -14,6 +14,7 @@
                 string_list += [letter + perm]
     return string_list
 
+
 def main():
     print(permute_string("abc"))
 
```